examples/env-vars/.npmrc

```diff
@@ -0,0 +1,2 @@
+## force pnpm to hoist 
+shamefully-hoist = true
```

Comment:
not needed, but also not against adding it here

Reply:
Ah, true. I copied our `minimal` starter over, so that's where this is from. Should be cleaned up in https://github.com/withastro/astro/pull/2455